composer.json

```diff
@@ -6,7 +6,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
+        "drupal/core-recommended": "^8.7",
```

Comment:
I wouldn't add this here, we can wait for 8.8 to do that?

Reply:
It goes back to 8.7 releases as well.